modules/users/client/components/AboutMe.component.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import sanitizeHtml from 'sanitize-html';
+import { withNamespaces } from '@/modules/core/client/utils/i18n-angular-load';
+{/* import { Trans } from 'react-i18next'; */}
```

Comment:
This can be removed.
(Also, the curly brackets around the comment are only needed inside a jsx/html block.)